### ISSUE TITLE
Regarding forecasting for many stations at a time

### ISSUE THREAD

**Issue by EnugulaVishnucharan** [thuml/TimeXer#25]
Hi authors
 I am working on time series data with exogenous features and found your paper TimeXer quite interesting .
 1) I have weather data of 500 stations (exogenous and endogenous features). With your TimeXer setting , I believe we can forecast for a single station at a time. I want to forecast at a time for all the stations . Your paper claim that forecasting is done for 3850 stations (large scale meteorology data)   . Is the forecasting done for each station at a time or all the stations at once .I went through your git but could not find . Please let me know if I am missing any file which has this information. 
  Is there a way possible to forecast all the stations at a time using exogenous features.
2) Please let me know if I can get pretrained model of TimeXer so that I can finetune the model on my data.
3) In the visualization used is it possible to indicate the forecasting data with timestamp on x axis.

**Comment by StraightSequoia** [comment]
Hi, there,

I have the same request as forecasting multiple stations with exogenous features. Please let me know if you have figured out how
